.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Can you include text for users to know that they can connect to us via slack too? Here is the template from [jazz core](https://raw.githubusercontent.com/tmobile/jazz/master/ISSUE_TEMPLATE.md) for reference.